extensions/parquet/table/src/main/java/io/deephaven/parquet/table/layout/ParquetFileHelper.java

```diff
@@ -4,15 +4,31 @@
 package io.deephaven.parquet.table.layout;
 
 import io.deephaven.parquet.table.ParquetTableWriter;
+import io.deephaven.util.channel.SeekableChannelsProvider;
+import org.jetbrains.annotations.NotNull;
 
+import java.net.URI;
 import java.nio.file.Path;
 
-final class ParquetFileHelper {
```

Comment:
Not sure it makes sense to expose this and method. Maybe just make as private method in ParquetTools?